src/App.js

```diff
@@ -1,25 +1,23 @@
-import logo from './logo.svg';
+import React, { Component } from 'react';
+import { Header } from './Header/Header.js';
+import { Switch, Route } from 'react-router-dom';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+export class App extends Component {
+  constructor() {
+    super()
+    this.state = {}
```

Comment:
I think I only need state here if I plan on adding the functionality to have multiple different decks. Otherwise the only state I should need can be in the Shuffler component.